src/components/buttons/RefreshButton/RefreshButton.tsx

```diff
@@ -25,27 +21,24 @@ export const RefreshButton = (props: IRefreshButtonProps) => {
 			setClicked(true);
 			// Set a timeout equal to the length of the spin animation/transition
 			setTimeout( () => setClicked(false), 550)
-			onClick(e);
+			onClick?.(e);
 		}
 	};
 
 	return (
-		<Container>
-			<button
+		<Container {...container}>
```

Comment:
So, this `<Container />` component handles disabled/ hover styles? Do you have types for the `container` props?

Reply:
`Container` is like our app-wide margin system, I think it's kind of fallen out of favor as people have left. But like `ITextButtonProps` extends `ILocalContainerProps` somewhere along the way. And actually since I changed it to use the `TextButton` I can remove that whole wrapper, since it gets drilled down to `ButtonBase`, which includes the `Container` wrapper. Good call!